src/errors.rs

```diff
@@ -8,10 +6,7 @@ use std::{
 /// This type is used to indicate errors that occur interpreting a `Ratchet`
 #[derive(Debug)]
 pub enum RatchetErr {
-    BadLen(usize),
-    BadEncoding(String),
     UnknownRelation,
-    Decode(DecodeError),
```

Comment:
I removed the string-based base64 encoding. I'm not aware of anyone using that & it seemed to be nontrivial to upgrade to a new internal ratchet state (with the salt).